src/app/layout/qgslayoutmapwidget.h

```diff
@@ -175,6 +175,29 @@ class QgsLayoutMapWidget: public QgsLayoutItemBaseWidget, private Ui::QgsLayoutM
 };
 
 
+class QgsLayoutMapItemBlocksLabelsModel : public QSortFilterProxyModel
+{
+    Q_OBJECT
+
+  public:
+
+    explicit QgsLayoutMapItemBlocksLabelsModel( QgsLayoutItemMap *map, QgsLayoutModel *layoutModel, QObject *parent = nullptr );
+
+    int columnCount( const QModelIndex &parent = QModelIndex() ) const override;
```

Comment:
Any reason why your not returning the static value of 1 in the header instead of the .cpp? (It's more about me understand your choice than questioning it really :wink: )

Reply:
Thanks master. ;)